=== command
CI="true" GITHUB_ACTIONS="true" GITHUB_ACTOR="some_test_username" GITHUB_API_URL="https://api.github.com" GITHUB_REPOSITORY="project_name/project_name" GITHUB_RUN_ID="35" GITHUB_SERVER_URL="https://github.com" GITHUB_WORKSPACE="/home/runner/work/actions-test/actions-test" GITHUB_REPOSITORY_ID="4" GITHUB_REPOSITORY_OWNER_ID="2" GITHUB_EVENT_NAME="pull_request" GITHUB_REF="refs/pull/123/merge" GITHUB_BASE_REF="main" GITHUB_HEAD_REF="some/branch-name" SEMGREP_PROJECT_CONFIG="tags:
- tag1
- tag_key:tag_val
" GITHUB_SHA="<MASKED>" GITHUB_EVENT_PATH="<MASKED>" SEMGREP_APP_TOKEN="fake-key-from-tests" SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<MASKED>" SEMGREP_VERSION_CACHE_PATH="<MASKED>" SEMGREP_ENABLE_VERSION_CHECK="0" SEMGREP_SEND_METRICS="off" semgrep ci --no-suppress-errors
=== end of command

=== exit code
2
=== end of exit code

=== stdout - plain
No mock address: GET https://semgrep.dev/c/auto
Traceback (most recent call last):
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/commands/wrapper.py", line 37, in wrapper
    func(*args, **kwargs)
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/commands/scan.py", line 630, in scan
    ) = semgrep.run_scan.run_scan(
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/run_scan.py", line 391, in run_scan
    configs_obj, config_errors = get_config(
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/config_resolver.py", line 1047, in get_config
    config, errors = Config.from_config_list(config_strs, project_url)
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/config_resolver.py", line 551, in from_config_list
    resolved_config = resolve_config(config, project_url)
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/config_resolver.py", line 479, in resolve_config
    config = parse_config_files(config_loader.load_config())
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/config_resolver.py", line 155, in load_config
    return [self._download_config()]
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/config_resolver.py", line 178, in _download_config
    return self._download_config_from_url(self._config_path)
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/config_resolver.py", line 198, in _download_config_from_url
    resp = app_session.get(url, headers={"Accept": "application/json"})
  File "/home/runner/.local/share/virtualenvs/cli-uZ-2zvZl/lib/python3.10/site-packages/requests/sessions.py", line 602, in get
    return self.request("GET", url, **kwargs)
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/app/session.py", line 190, in request
    response = super().request(*args, **kwargs)
  File "/home/runner/.local/share/virtualenvs/cli-uZ-2zvZl/lib/python3.10/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/runner/.local/share/virtualenvs/cli-uZ-2zvZl/lib/python3.10/site-packages/requests_mock/mocker.py", line 185, in _fake_send
    return _original_send(session, request, **kwargs)
  File "/home/runner/.local/share/virtualenvs/cli-uZ-2zvZl/lib/python3.10/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/runner/.local/share/virtualenvs/cli-uZ-2zvZl/lib/python3.10/site-packages/requests_mock/adapter.py", line 261, in send
    raise exceptions.NoMockAddress(request)
requests_mock.exceptions.NoMockAddress: No mock address: GET https://semgrep.dev/c/auto

=== end of stdout - plain

=== stderr - plain

=== end of stderr - plain